src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecution.tsx

```diff
@@ -12,6 +12,7 @@ import { useTranslate } from 'react-polyglot'
 import { useErrorHandler } from 'hooks/useErrorToast/useErrorToast'
 import { useWallet } from 'hooks/useWallet/useWallet'
 import { MixPanelEvent } from 'lib/mixpanel/types'
+import { TRADE_POLL_INTERVAL_MILLISECONDS } from 'lib/swapper/swappers/LifiSwapper/LifiSwapper'
```

Comment:
Perhaps for clarity - pretty obvious this is LiFi-specific when if you read the import, but not obvious if you only read the consumption

```suggestion
import { TRADE_POLL_INTERVAL_MILLISECONDS as LIFI_TRADE_POLL_INTERVAL_MILLISECONDS } from 'lib/swapper/swappers/LifiSwapper/LifiSwapper'
```

Reply:
Went with rename of const variable